Makefile

```diff
@@ -40,5 +48,6 @@ IDIRS	+=	-I lib/util
 
 CFLAGS	+=	-g
 CFLAGS	+=	${IDIRS}
+CFLAGS  += -DCERTFILE
```

Comment:
What do you think this does, exactly?

Reply:
https://gcc.gnu.org/onlinedocs/gcc/Preprocessor-Options.html

Reply:
Oh lol, I'm missing the assignment